docs/history/changelog.md

```diff
@@ -54,7 +54,7 @@ Per the [normative and non-normative content and changes policy](https://docs.go
 * [#1218](https://github.com/open-contracting/standard/pull/1218) Add the 'COFOG' code from the organization classifiation extension to the `itemClassificationScheme` codelist.
 
 ### Schema
-
+* [#1229](https://github.com/open-contracting/standard/pull/1229) Standardize the descriptions of `Planning.Documents`, `Tender.Documents`, `Award.Documents`, `Contract.Documents`, and `Contract.Implementation.Documents`.
```

Comment:
```suggestion
* [#1229](https://github.com/open-contracting/standard/pull/1229) Standardize the descriptions of `planning.documents`, `tender.documents`, `awards.documents`, `contracts.documents`, and `contracts.implementation.documents`.
```

Reply:
Shouldn't it be award and contract, not awards and contracts? (Sorry for the capitalization.)

Reply:
The JSON path is plural. The definition in the JSON Schema is singular and capitalized. The data uses the JSON paths, so we prefer the uncapitalized, plural versions.